<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register</class>
 <widget class="QWidget" name="register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(78, 131, 181);</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>471</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
border : 1px solid black;
border-radius : 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>380</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
border : 2px solid grey;
border-radius : 10px;
font-size : 14px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
color: black;
font-weight : bold;</string>
     </property>
     <property name="text">
      <string>Nama</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size : 24px;
font-weight : bold;
border-bottom : 2px solid white;</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>410</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
color: black;
font-weight : bold;</string>
     </property>
     <property name="text">
      <string>Tanggal Lahir</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
border : 2px solid grey;
border-radius : 10px;
font-size : 14px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>430</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
border : 2px solid grey;
border-radius : 10px;
font-size : 14px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>360</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
color: black;
font-weight : bold;</string>
     </property>
     <property name="text">
      <string>Nomor HP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
color: black;
font-weight : bold;</string>
     </property>
     <property name="text">
      <string>Alamat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
border : 2px solid grey;
border-radius : 10px;
font-size : 14px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>480</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
border : 2px solid grey;
border-radius : 10px;
font-size : 14px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>330</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
border : 2px solid grey;
border-radius : 10px;
font-size : 14px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
color: black;
font-weight : bold;</string>
     </property>
     <property name="text">
      <string>NIK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>530</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(78, 131, 181);
font-weight : bold ;
border-radius : 10px;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
border : 2px solid grey;
border-radius : 10px;
font-size : 14px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>455</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
color: black;
font-weight : bold;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : #f5f5f5;
color: black;
font-weight : bold;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
